src/main/resources/repairability/elixir/ep_1.xml

```diff
@@ -0,0 +1,6 @@
+<pattern name="any_statement_deep">
```

Comment:
name of all xml are same, ie, `any_statement_deep`.
File and pattern_name renaming should look cool

Reply:
Thank you for your comments Siddharth.